app/models/find_api/course.rb

```diff
@@ -15,7 +15,7 @@ def self.fetch(provider_code, course_code)
         .first
     rescue JsonApiClient::Errors::NotFound
       nil
-    rescue JsonApiClient::Errors::ServerError
+    rescue JsonApiClient::Errors::ServerError, JsonApiClient::Errors::ConnectionError
```

Comment:
I've searched a bit but can't figure out how to WebMock a "connection refused" so I can write a spec this case. If anyone reads this comment and knows, please reply and I'll open a follow-up PR to address the gap in our specs. 👍